_analyzers/token-filters/classic.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: default
+title: classic
```

Comment:
Is this intentionally lowercase?

Reply:
Should be uppercase, thanks for catching!